cfme/ansible/__init__.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
Why is this file named as it is and located where it is?

Although its written under the context of an ansible test with a file, its quite abstract, and looks more like some generic utility class.

There are already `SSHClient.get_file`, `put_file`, and `patch_file` methods. I think these methods should be added to `SSHClient`, not defined separately.

Reply:
Mike we had that methods separately as we try to avoid methods like `remove file` check `availability`, personal validation `context manager` as part of Client.

but yup, I need two methods which i moved to Client... adding context manager in client i think not good thing so added `wait_for` in test it self.